src/modules/login/Login.tsx

```diff
@@ -8,27 +8,34 @@ import { useAuthProvider } from 'core/auth';
 
 import LoginForm from './login-form';
 
+import Navbar from "shared/components/navbar";
```

Comment:
You can import like { Navbar, Footer } from shared/components. Just add exports in shared/components/index.ts file - remember about alphabetic order